File: example/src/main/java/com/example/errorview/Example2Activity.java
package com.example.errorview;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.widget.Toolbar;
import android.view.MenuItem;
import android.widget.FrameLayout;

import com.hendraanggrian.widget.ErrorView;

import butterknife.BindView;

/**
 * @author Hendra Anggrian (hendraanggrian@gmail.com)
 */
public class Example2Activity extends BaseActivity {

    @BindView(R.id.toolbar_example) Toolbar toolbar;
    @BindView(R.id.framelayout_example) FrameLayout frameLayout;

    @Override
    public int getContentView() {
        return R.layout.activity_example;
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setSupportActionBar(toolbar);
        ErrorView.make(frameLayout, "You have no new emails", ErrorView.LENGTH_INDEFINITE)
                .setBackdrop(R.drawable.bg_empty)
                .setContentMarginBottom((int) getResources().getDimension(R.dimen.example2_content_margin))
                .show();
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        if (item.getItemId() == android.R.id.home)
            finish();
        return super.onOptionsItemSelected(item);
    }
}

File: example/src/main/java/com/example/errorview/Example2Activity.kt
package com.example.errorview

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.MenuItem
import com.hendraanggrian.kota.content.toPx
import com.hendraanggrian.widget.ErrorView2
import com.hendraanggrian.widget.errorView
import kotlinx.android.synthetic.main.activity_example.*

/**
 * @author Hendra Anggrian (hendraanggrian@gmail.com)
 */
class Example2Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_example)
        setSupportActionBar(toolbar)
        frameLayout.errorView("You have no new emails", ErrorView2.LENGTH_INDEFINITE)
                .setBackdropDrawable(R.drawable.bg_empty)
                .setLogoDrawable(null)
                .setContentMarginBottom(64.toPx())
                .show()
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            finish()
        }
        return super.onOptionsItemSelected(item)
    }
}
